File: Front-end/SIRCADE/src/app/schedules_programming/interfaces/responses/schedule-programming-info.response.ts
export interface ScheduleProgrammingInfoResponse {
  id: number;
  sportFieldId: number;
  sportFieldName: string;
  clientId?: number;
  clientName: string;
  type: number;
  typeName: string;
  startDate: Date;
  endDate: Date;
  comment?: string;
  registerUser: string;
  lightColor: string;
  darkColor: string;
}

File: Back-end/SIRCADE.ApiCore.Controllers/SIRCADE.ApiCore.Controllers/SchedulesProgramming/Responses/ScheduleProgrammingInfoResponse.cs
namespace SIRCADE.ApiCore.Controllers.SchedulesProgramming.Responses;

public record ScheduleProgrammingInfoResponse(
    int Id,
    int SportFieldId,
    string SportFieldName,
    int? ClientId,
    string? ClientName,
    DateTime StartDate,
    DateTime EndDate,
    string? Comment,
    string RegisterUser,
    int Type,
    string TypeName,
    string LightColor,
    string DarkColor);